Tests/AsyncHTTPClientTests/HTTPConnectionPool+HTTP2StateMachineTests.swift

```diff
@@ -1146,11 +1151,12 @@ class HTTPConnectionPool_HTTP2StateMachineTests: XCTestCase {
             idGenerator: .init(),
             maximumConcurrentHTTP1Connections: 8,
             retryConnectionEstablishment: true,
+            preferHTTP1: false,
             maximumConnectionUses: nil
         )
 
         var connectionIDs: [HTTPConnectionPool.Connection.ID] = []
-        for el in [el1, el2, el2] {
+        for el in [el1, el2] {
```

Comment:
why did you remove `el2` here?

Reply:
This is because we now directly start with h2 rather than h1 and then migrating to h1. 
h2 has checks to not allow creating connections for same EL. The second el2 just fails the test because the check fails.